components/newsletter-sign-up/newsletter-sign-up.jsx

```diff
@@ -48,6 +24,8 @@ class NewsletterSignUp extends React.Component {
   // state update function
   apiSubmissionFailure(e) {
```

Comment:
let's rename this to `err` or `error`. Also, it feels like if this arg exists, it's going to be an error because that's how we call it. Is there a possibility of it getting called where there is an argument, but it's not an error?